Files/Views/Pages/PropertiesDetails.xaml

```diff
@@ -1,486 +1,611 @@
-                    x:Name="Icon2"
+                    x:Name="Icon"
                     Grid.Row="0"
                     Grid.Column="0"
-                    Width="110"
+                    Width="80"
                     Height="80"
-                    Margin="0,7"
+                    Margin="{StaticResource PropertyNameMargin}"
                     HorizontalAlignment="Stretch"
                     VerticalAlignment="Stretch">
-                    <Image
-                        x:Name="FolderGlyphIcon2"
-                        Width="70"
-                        Height="70"
-                        HorizontalAlignment="Stretch"
-                        VerticalAlignment="Stretch"
-                        x:Load="{x:Bind ViewModel.LoadFolderGlyph, Mode=OneWay}"
-                        Stretch="Uniform">
-                        <Image.Source>
-                            <SvgImageSource
-                                RasterizePixelHeight="128"
-                                RasterizePixelWidth="128"
-                                UriSource="{x:Bind ViewModel.FolderIconSource}" />
-                        </Image.Source>
-                    </Image>
-                    <FontIcon
-                        x:Name="EmptyImageIcon2"
-                        HorizontalAlignment="Left"
-                        VerticalAlignment="Stretch"
-                        x:Load="{x:Bind ViewModel.LoadUnknownTypeGlyph, Mode=OneWay}"
-                        FontFamily="{StaticResource FluentUIGlyphs}"
-                        FontSize="70"
-                        Glyph="&#xea00;" />
-                    <FontIcon
-                        x:Name="WebShortcutGlyph2"
-                        HorizontalAlignment="Left"
-                        VerticalAlignment="Stretch"
-                        x:Load="{x:Bind ViewModel.LoadLinkIcon, Mode=OneWay}"
-                        FontFamily="{StaticResource FluentUIGlyphs}"
-                        FontSize="70"
-                        Glyph="&#xEAAB;" />
-                    <Image
-                        x:Name="itemIcon2"
-                        HorizontalAlignment="Stretch"
-                        VerticalAlignment="Stretch"
-                        x:Load="{x:Bind ViewModel.LoadFileIcon, Mode=OneWay}"
-                        Source="{x:Bind ViewModel.FileIconSource, Mode=OneWay}" />
+                    <usercontrols:FileIcon
+                        ItemSize="70"
+                        ViewModel="{x:Bind ViewModel}" />
                 </Grid>
                 <TextBox
-                    x:Name="itemFileName2"
+                    x:Name="ItemFileName"
                     x:Uid="PropertiesItemFileName"
                     Grid.Row="0"
                     Grid.Column="1"
-                    Margin="20,0"
-                    HorizontalAlignment="Stretch"
-                    VerticalAlignment="Center"
-                    BorderThickness="1"
-                    IsReadOnly="True"
```

Comment:
There is no logic in the Details Tab to rename the file. This is currently only possible in the General tab.
Until it's possible to actually save the new file name from the Details tab this textbox should remain "Read-Only"